docs/source/02_installation.md

```diff
@@ -49,15 +49,15 @@ to have the Spark binaries on your system; prebuilt binaries can be downloaded f
 https://amplab.cs.berkeley.edu/jenkins/job/ADAM/) builds ADAM against Spark versions 1.6.1 and 2.0.0,
 Scala versions 2.10 and 2.11, and Hadoop versions 2.3.0 and 2.6.0.
 
-Once this alias is in place, you can run ADAM by simply typing `adam-submit` at the commandline, e.g.
+Once this alias is in place, you can run ADAM by simply typing `adam-submit` at the commandline.
 
 ```bash
 $ adam-submit
 ```
 
-## Building for Python {#python-build}
+## Building for Python
```

Comment:
Explanation for anchor tags provided in earlier comment. 

For the anchor link, I thought it was for an outdated anchor that had been removed, there's no `#python` anchor. When you click on that link on [this](http://adam.readthedocs.io/en/latest/source/02_installation/) page, it does not take you anywhere.